src/driver.ts

```diff
@@ -278,7 +250,18 @@ export const getMinimumTemperatureInKelvinForDevice = (device: Device): number =
  * @returns {number} The maximum temperature in Kelvin supported by the device
  */
 export const getMaximumTemperatureInKelvinForDevice = (device: Device): number => {
-  return MAXIMUM_TEMPERATURE_IN_KELVIN_BY_DEVICE_TYPE[device.type];
+  const allowedTemperatures = ALLOWED_TEMPERATURES_IN_KELVIN_BY_DEVICE_TYPE[device.type];
+  return allowedTemperatures[allowedTemperatures.length - 1];
+};
+
+/**
+ * Gets all temperature values in Kelvin supported by a device
+ *
+ * @param {Device} device The device to check the allowed temperatures for
+ * @returns {number[]} The temperature values in Kelvin supported by the device
+ */
+const getAllowedTemperaturesInKelvinForDevice = (device: Device): number[] => {
```

Comment:
I did consider exporting this function - happy to add it later if people want it!